### DISCUSSION TITLE
Manual or drag and drop sorting of entries

### DISCUSSION THREAD

**Discussion by martinsagan** [keepassxreboot/keepassxc#8679] [Ideas]
Hello KeePassXC team,

could you please add function to manual sorting of entries or drag and drop sorting in entries section? 
Now it's works only in left menu. I use by default "no-sorting" in entries section and if I want change position of any entry I must run KeePass 2 classic and change the positions of entry here :) (workaround)

Thanks, best regards
Martin

**Comment by droidmonkey** [comment]
Click the header of any column in the entry view until the sorting arrow disappears. Now you can move entries up and down using the context menu or keyboard shortcut.

**Reply by martinsagan** [reply]
Thanks a lot 😊 You made my day 👍
Unless I had disabled sorting, I didn't see that option in the context menu.
